contracts/src/interfaces/IOmniAVS.sol

```diff
@@ -24,11 +25,13 @@ interface IOmniAVS {
     /**
      * @notice Struct representing an OmniAVS operator
      * @custom:field addr       The operator's ethereum address
+     * @custom:field pubkey     The operator's 64 byte uncompressed secp256k1 public key
      * @custom:field delegated  The total amount delegated, not including operator stake
      * @custom:field staked     The total amount staked by the operator, not including delegations
      */
     struct Operator {
         address addr;
+        bytes pubkey;
         uint96 delegated;
         uint96 staked;
     }
```

Comment:
We can remove `addr` in a separate PR if we decide we don't want it. It's not strictly needed.